statistics.txt

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Odd that this is here, I think you need to update your .gitignore to include data files
TO BE CHANGED

Reply:
not resolved.

Reply:
can @Meg1456 untrack the file